server/src/db/migrations/20210227184541-create-infrastructure-requirement.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('infrastructureRequirements', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING,
+        defaultValue: Sequelize.literal('uuid_generate_v4()'),
```

Comment:
If you are going to continue using the upsert pattern, this may be an unnecessary